### PR TITLE
[mono] Update to get fixes from xplat-master

### PR DESCRIPTION
**Pull request by radical**

### CODE REVIEW

**Comment by radical** [comment]
@kg fyi, this mono 2019-02 seems to be using roslyn `2.8.2.62916 (2ad4aabc)` (from `csc -version`), I removed the roslyn version bump in msbuild. This would mean that it will run and test with `2.8.0`, but the final install will be with whatever mono has. If you know the correct nuget version then please feel free to update for the branch.

**Comment by kg** [comment]
This seems right to me but I have no idea how to identify the right roslyn package. If it works then we're fine, I guess?

**Comment by radical** [comment]
Yeah, I think we should be ok for this one. I will try to figure out better ways for getting this in sync. Thanks!

**Comment by radical** [comment]
@kg The step in your update guide:
`4. Find that revision in the 'roslyn compilers' nuget feed: https://dotnet.myget.org/feed/roslyn/package/vsix/7922692f-f018-45e7-8f3f-d3b7c0262841`
.. shouldn't this be the nuget version that we want to use? If so, then the nuget version from which you get the binaries can be added to the commit message.

File: eng/cibuild_bootstrapped_msbuild.sh [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
```
Comment (lewing) [inline eng/cibuild_bootstrapped_msbuild.sh]: do you really want to revert this part?

**Review by lewing — COMMENTED** [review]
The actual changes here look fine @kg should we wait on the roslyn bump?

**Comment by kg** [comment]
If this fixes something, I'd say do it now. We're only updating the bump on 04 anyway

**Review by lewing — APPROVED** [review]

**Pull request merged**